src/containers/VisitPage.js

```diff
@@ -70,8 +70,8 @@ class VisitPage extends Component {
 
   componentDidMount() {
     const data = { client: this.state.client };
-    const confirm = (message, callback) =>
-      this.prompt(callback, data, message);
+    const confirm = (message, callback, leaveText, header) =>
```

Comment:
This won't cause a problem, but it isn't technically right. It is React Router, or more specifically [history](https://github.com/ReactTraining/history), that will be calling this function. So we can't change its method signature.

The default confirm function is [here](https://github.com/ReactTraining/history/blob/master/modules/DOMUtils.js#L7).